### DISCUSSION TITLE
Recommended install path for debian-based systems?

### DISCUSSION THREAD

**Discussion by wez** [kata-containers/kata-containers#9839] [Q&A]
I'm interested in trying kata-containers but I'm unsure how best to get started on my ubuntu dev environment because the [docs](https://github.com/kata-containers/kata-containers/tree/main/docs/install#official-packages) are centos/fedora-centric.

[kata-manager](https://github.com/kata-containers/kata-containers/blob/main/utils/README.md)'s docs are phrased in a way that makes me not want to install it on anything other than a throwaway system, because it is not clear that the installation is reversible.

I can see from googling that there used to be a solution for ubuntu, and in fact, my system is already running a kata-containers 2.4 snap (I have no idea what caused that to be installed!), but I can also see that that is unmaintained and old.

What I want to do is experiment with kata and docker on this system so that I can evaluate whether I want to deploy it to my private infrastructure, which is debian-based.

What's the current state/recommendations for users of debian-based systems?

**Comment by mikemma** [comment]
I see there is this info for ubuntu: https://github.com/kata-containers/documentation/blob/master/install/ubuntu-installation-guide.md but the key http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/${BRANCH}/xUbuntu_$(lsb_release -rs)/Release.key is missing.
So, I am also unable to install kata-containers in ubuntu.
